### ISSUE TITLE
Can't override site_name setting

### ISSUE THREAD

**Issue by RomainFallet** [Mark-H/ClientConfig#82]
On Modx 2.3.1-pl, I can't override "site_name" setting simply by setting "site_name" key on client config.
Is there something I'm doing wrong ?
[Edit] It seems that it does not override it, only in the manager, for example, on the dashboard. If I use the setting in my client view, it works fine.

**Comment by vallev** [comment]
I tried to deal with this too with no luck, but I've found an easy way to completely change the setting you wish to. Let's assume you have option called "site_name" in ClientConfig. Rename it to something like "website_name". Now you just have to add the following code inside your ClientConfig plugin:
```
...
$newname = $modx->getOption('website_name');
$sitename = $modx->getObject('modSystemSetting', 'site_name');
$sitename->set('value', $newname);
$sitename->save();
...
break;
```
Now every time when you change "website_name" inside ClientConfig, the real "site_name" setting updates as well. That's it!
